adam/language_specific/chinese/chinese_phase_1_lexicon.py

```diff
@@ -0,0 +1,140 @@
+"""We use Yale romanization in this code to maintain UTF-8 encoding;
+spaces indicate borders between Chinese characters.
+
+Currently, mass nouns are specified despite this not being realised in
+the same way in Chinese as in English.
+
+We don't provide plural specifications for nouns since this isn't a
+morphologically-realized feature in Chinese.
+"""
+from adam.language_specific import (
+    FIRST_PERSON,
+    SECOND_PERSON,
+    MASS_NOUN,
+    ALLOWS_DITRANSITIVE,
+)
+from adam.language.dependency.universal_dependencies import (
+    ADJECTIVE,
+    NOUN,
+    PROPER_NOUN,
+    VERB,
+)
+from adam.language.lexicon import LexiconEntry
+from adam.language.ontology_dictionary import OntologyLexicon
+from adam.ontology.phase1_ontology import (
+    BABY,
+    BALL,
+    BIRD,
+    BLACK,
+    BLUE,
+    BOOK,
+    BOX,
+    CAR,
+    CHAIR,
+    COME,
+    COOKIE,
+    CUP,
+    DAD,
+    DOG,
+    DOOR,
+    DRINK,
+    EAT,
+    FALL,
+    FLY,
+    GAILA_PHASE_1_ONTOLOGY,
+    GIVE,
+    GO,
+    GREEN,
+    GROUND,
+    HAND,
+    HAS,
+    HAT,
+    HEAD,
+    HOUSE,
+    JUICE,
+    JUMP,
+    MILK,
+    MOM,
+    MOVE,
+    PUSH,
+    PUT,
+    RED,
+    ROLL,
+    SIT,
+    SPIN,
+    TABLE,
+    TAKE,
+    THROW,
+    TRANSPARENT,
+    TRUCK,
+    WATER,
+    WHITE,
+    LIGHT_BROWN,
+    DARK_BROWN,
+)
+
+ME = LexiconEntry(
+    "wo3",
+    NOUN,
+    plural_form="wo3 men",
+    intrinsic_morphosyntactic_properties=[FIRST_PERSON],
+)
+YOU = LexiconEntry(
+    "ni3",
+    NOUN,
+    plural_form="ni3 men",
```

Comment:
We should probably omit the plural forms of these, since I think they are realized syntactically and not morphologically.

Reply:
Just pushed a new version to fix that